content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst

```diff
@@ -2,40 +2,122 @@
-to `30`. Once the value is entered, click :guilabel:`Save` to apply the setting to the location.
-Now, once an inventory adjustment is applied to this location, the next scheduled count date will
-be automatically set based on the number of days in the :guilabel:`Inventory Frequency (Days)`
-setting.
+From this page, click into a location to reveal the settings and configuration page for that
+location. Click :guilabel:`Edit` to edit the location settings.
 
-.. image:: cycle_counts/inventory-frequency.png
+Under the :guilabel:`Cyclic Inventory` section, locate the :guilabel:`Inventory Frequency (Days)`
+field, which should be set to `0` (if this location has not been edited previously). In this field,
+change the value to whichever number of days is desired.
+
+.. image:: cycle_counts/cycle-counts-location-frequency.png
    :align: center
-   :alt: Edit a location to change the inventory frequency.
+   :alt: Location frequency setting on location.
+
+.. example::
+   A location that needs an inventory count every 30 days should have the :guilabel:`Inventory
+   Frequency (Days)` value set to `30`.
+
+Once the frequency has been changed to the desired number of days, click :guilabel:`Save` to save
+changes. Now, once an inventory adjustment is applied to this location, the next scheduled count
+date will be automatically set based on the value entered into the :guilabel:`Inventory Frequency
+(Days)` field.
+
+Count inventory by location
+===========================
+
+To perform a cycle count for a specific location in the warehouse, navigate to
+:menuselection:`Inventory app --> Operations --> Inventory Adjustments`. This reveals an
+:guilabel:`Inventory Adjustments` page containing all products currently in stock, with each product
+listed on its own line.
+
+From this page, the :guilabel:`Filters` and :guilabel:`Group By` buttons (at the top of the page,
+under the :guilabel:`search bar`), can be used to select specific locations and perform inventory
+counts.
+
+.. image:: cycle_counts/cycle-counts-inventory-adjustments-page.png
+   :align: center
+   :alt: Inventory adjustments page.
+
+To select a specific location and view all products within that location, click :guilabel:`Group
+By`, then click :guilabel:`Add Custom Group` to reveal a new drop-down menu to the right. Click
+:guilabel:`Location` from the drop-down menu, then click :guilabel:`Apply`. The page now displays
+condensed drop-down menus of each location in the warheouse that has products in stock, and a cycle
+count can be performed for all products in that location.
+
+.. tip::
+   In large warehouses with a large number of locations and high volume of products, it might be
+   easier to search for the specific location desired. To do this, from the
+   :menuselection:`Inventory Adjustments` page, click :guilabel:`Filters`. Then, click
+   :guilabel:`Add Custom Filter` to reveal a new menu to the right. Click this menu to reveal three
+   drop-downs.
```

Comment:
```suggestion
   In large warehouses, with a large number of locations and high volume of products, it might be
   easier to search for the specific location desired. To do this, from the :guilabel:`Inventory
   Adjustments` page, click :guilabel:`Filters`. Then, click :guilabel:`Add Custom Filter` to
   reveal a new menu to the right. This menu contains three drop-downs.
```
changed it to a guilabel for the same reason mentioned above